src/kw_remote.sh

```diff
@@ -188,6 +182,12 @@ function add_new_remote()
     printf '  Port %s\n' "${remote_parameters['REMOTE_PORT']}"
     printf '  User %s\n' "${remote_parameters['REMOTE_USER']}"
   } >> "$remote_config_file"
+
+  # Check if user request to set this new entry a default
+  if [[ -z "$first_time" && ${options_values['DEFAULT_REMOTE_USED']} -eq 1 ]]; then
+    options_values['DEFAULT_REMOTE']="$name"
+    set_default_remote
+  fi
```

Comment:
When you set a new remote for the first time, the file `remote.config` might not be available, and the highlighted code will fail because the target file does not exist. That's why I moved this file below to the file creation.